src/Wallabag/CoreBundle/Controller/EntryController.php

```diff
@@ -250,6 +250,10 @@ private function showEntries($type, Request $request, $page)
         $entries->setMaxPerPage($this->getUser()->getConfig()->getItemsPerPage());
         $entries->setCurrentPage($page);
 
+        if ($type === 'unread' && count($entries) == 0) {
```

Comment:
Well, I'll prefer to see that in `showUnreadAction` with a test on the route and an other query to retrieve the count on **all** entries (not only the unread one)